DSCResources/MSFT_xIPAddress/MSFT_xIPAddress.psm1

```diff
@@ -185,102 +228,108 @@ function Test-Properties
         if(-not (Get-NetIPInterface -InterfaceAlias $InterfaceAlias -AddressFamily `
             $AddressFamily).Dhcp.ToString().Equals('Disabled'))
         {
-            Write-Verbose -Message "DHCP is NOT disabled."
+            Write-Verbose -Message ( @( "$($MyInvocation.MyCommand): "
+                'DHCP is NOT disabled.'
+                ) -join '' )
             $requiresChanges = $true
         }
         else
         {
-            Write-Verbose -Message "DHCP is already disabled."
+            Write-Verbose -Message ( @( "$($MyInvocation.MyCommand): "
+                'DHCP is already disabled.'
+                ) -join '' )
         }
+    } catch {
```

Comment:
Do we really think try/catch is needed here or is the other places we throw better?